### PR TITLE
Fix missing fields + renaming

### PR DESCRIPTION
**Pull request by leogrim**
@yingjieMiao I noticed a couple indexed fields were missing / wrong (see discoverable case).
Also renamed org-related fields in `KeepIndexable` to be consistent with the other ones, and the ones in `LibraryIndexable` to be consistent with `KeepIndexable`.

### CODE REVIEW

File: kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala [hunk 1]
```diff
@@ -19,6 +19,8 @@ object KeepFields {
   val userField = "user"
   val userIdField = "userId"
   val userDiscoverableField = "userDisc"
+  val orgField = "org"
+  val orgIdField = "orgId"
```
Comment (leogrim) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala:23]: Renaming and moving them up the list to be consistent.

File: kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala [hunk 2]
```diff
@@ -50,10 +50,19 @@ case class KeepIndexable(keep: Keep, tags: Set[Hashtag], shard: Shard[Normalized
...
+      doc.add(buildKeywordField(uriDiscoverableField, keep.uriId.toString))
+      doc.add(buildKeywordField(userDiscoverableField, keep.userId.toString))
+    }
+
+    keep.organizationId.foreach { orgId =>
+      doc.add(buildKeywordField(orgField, orgId.toString))
+      if (keep.visibility == LibraryVisibility.PUBLISHED || keep.visibility == LibraryVisibility.ORGANIZATION) {
+        doc.add(buildKeywordField(orgDiscoverableField, keep.organizationId.get.id.toString))
+      }
+    }
```
Comment (leogrim) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala:65]: That logic needs to be updated with the new `LibraryVisibility.Organization`
Also, `orgField` was missing :)

File: kifi-backend/modules/search/app/com/keepit/search/index/graph/library/LibraryIndexable.scala [hunk 3]
```diff
@@ -125,13 +125,15 @@ class LibraryIndexable(library: DetailedLibraryView) extends Indexable[Library,
     }
 
     doc.add(buildKeywordField(ownerField, library.ownerId.id.toString))
+    library.orgId.foreach { orgId =>
+      doc.add(buildKeywordField(orgField, orgId.id.toString))
+    }
```
Comment (leogrim) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/library/LibraryIndexable.scala:130]: `orgField` was missing here too.

File: kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala [hunk 4]
```diff
@@ -83,16 +92,14 @@ case class KeepIndexable(keep: Keep, tags: Set[Hashtag], shard: Shard[Normalized
 
     doc.add(buildIdValueField(uriIdField, keep.uriId))
     doc.add(buildIdValueField(userIdField, keep.userId))
-    keep.libraryId.foreach(libId => doc.add(buildIdValueField(libraryIdField, libId)))
+    doc.add(buildIdValueField(libraryIdField, keep.libraryId.get))
+    doc.add(buildIdValueField(orgIdField, keep.organizationId.getOrElse(Id[Organization](-1))))
```
Comment (leogrim) [inline kifi-backend/modules/search/app/com/keepit/search/index/graph/keep/KeepIndexable.scala:96]: Grouping related fields and calling `keep.libraryId.get` to make our assumption that it's there explicit.

**Comment by yingjieMiao** [comment]
good catches. thanks!

**Pull request merged**